app/middleware/selectionValidator.js

```diff
@@ -0,0 +1,12 @@
+const messages = require('../lib/messages');
+const routeHelper = require('./routeHelper');
+
+function sympmtomsSelection(req, res, next) {
+  if (!res.locals.symptoms) {
+    routeHelper.renderSymptomsPage(req, res, messages.mandatorySelectionMessage());
```

Comment:
The message doesn't need passing through here, see the comment made against `routeHelper.js`.